### PR TITLE
Rss feed

### PR DESCRIPTION
**Pull request by msamgan**
PR for issue #1334

### CODE REVIEW

**Comment by msamgan** [comment]
Once you get this one merged, if you merge. I will use this as a base for the sitemap.xml issue.

**Comment by msamgan** [comment]
please close this PR.
Merge or Reject...

File: resources/views/feed.blade.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        <link><![CDATA[ {{ url("/feed") }} ]]></link>
+        <description><![CDATA[ {{ config("canvas.description") }} ]]>
+        </description>
+        <language>{{ app()->getLocale() }}</language>
+        <pubDate>{{ now() }}</pubDate>
+        <lastBuildDate>{{ now() }}</lastBuildDate>
+        @foreach($posts as $post)
+            <item>
+                <title><![CDATA[{{ $post->title }}]]></title>
+                <link>{{ url($post->slug) }}</link>
```
Comment (austintoddj) [inline resources/views/feed.blade.php:16]: Passing in the post slug is good, but we should be using the absolute path to the post - which can vary. Depending if someone wants their publication at: /blog, or /news, etc. 

You could try to use [this config value](https://github.com/austintoddj/canvas/blob/develop/config/canvas.php#L42), but there could still be edge cases.

Comment (msamgan) [reply resources/views/feed.blade.php:16]: we can have two approaches.

1. we can ask the user to add the static path of the post in the config.
2. we can make the feed view publishable, so if the user has a custom url he can configure the feed accordingly.

or we can do both.

what do you say about that ?

Comment (austintoddj) [reply resources/views/feed.blade.php:16]: > we can ask the user to add the static path of the post in the config

I don't think specifying a static path per post is a good UX. 

> we can make the feed view publishable...configure the feed accordingly

Still a heavier lift than what I'd expect as a user. Essentially, I'd like to see it where you can provide a config value in your .env file - we'd also need to take into account other taxonomies (tags/topics) - and then a switch where you can either enable or disable the feature.

**Pull request closed without merging**